File: model/FantasyBook.php
<?php

class FantasyBook extends Book{
    private $theme;

    //ctor
    public function FantasyBook(&$rDate, &$iDate, &$price, &$isbn, &$id, &$title, &$publisher, &$author, &$theme){
        parent::__construct($rDate, $iDate, $price, $isbn, $id, $title, $publisher, $author);
        $this->theme = $theme;
    }

    //getter and setter for theme

    public function getTheme() {
        return $this->theme;
    }
    public function setTheme($theme) {
        $this->theme = $theme;
    }
}
?>

File: FantasyBook.java
import java.util.Date;

public class FantasyBook extends Book{
    private String theme;

    public FantasyBook(){super();} //default ctor

    public FantasyBook(Date rDate, Date iDate, double price, int isbn, 
    int id, String title, Publisher publisher, Author author, String theme){ //ctor
        super(rDate, iDate, price, isbn, id, title, publisher, author);
        this.theme = theme;
    }

    //getter and setter for theme

    public String getTheme() {
        return theme;
    }
    public void setTheme(String theme) {
        this.theme = theme;
    }
}